compression/cutils/tests/utils/helpers.ts

```diff
@@ -0,0 +1,284 @@
+import fs from "fs";
+import path from "path";
+import { Connection, Keypair, LAMPORTS_PER_SOL, PublicKey } from "@solana/web3.js";
+
+// define some default locations
+const DEFAULT_KEY_DIR_NAME = ".local_keys";
+const DEFAULT_PUBLIC_KEY_FILE = "keys.json";
+const DEFAULT_DEMO_DATA_FILE = "demo.json";
+
+/*
+  Load locally stored PublicKey addresses
+*/
+export function loadPublicKeysFromFile(
+    absPath: string = `${DEFAULT_KEY_DIR_NAME}/${DEFAULT_PUBLIC_KEY_FILE}`,
+) {
+    try {
+        if (!absPath) throw Error("No path provided");
+        if (!fs.existsSync(absPath)) throw Error("File does not exist.");
+
+        // load the public keys from the file
+        const data = JSON.parse(fs.readFileSync(absPath, { encoding: "utf-8" })) || {};
+
+        // convert all loaded keyed values into valid public keys
+        for (const [key, value] of Object.entries(data)) {
+            data[key] = new PublicKey(value as string) ?? "";
+        }
+
+        return data;
+    } catch (err) {
+        // console.warn("Unable to load local file");
```

Comment:
Why not log the error? Empty try catches are often causing unnecessary debugging